### ISSUE TITLE
Content search portlet is not working if you change the system language to Spanish 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#17797]
We have problems trying to use the content search portlet when you change the default language to Spanish. We are not showing contents here and also we have errors in the JS console. 

The same thing in all the filtered portlets that are based on the content search. 

**FYI: Tested with the other languages and works as expected, the problem is just with the Spanish**

SC: https://www.screencast.com/t/VgQsZW8l

### Steps to reproduce the behavior:
1. In the login screen, change the default language to Spanish 
2. Login as admin
3. Go to the content search 
4. Try to click on advanced

### Expected behavior
We need be able to work normally in any of the supported languages 

### Desktop (please complete the following information):
Tested on release-5.2.3 // Oracle // FF

**Comment by john-thomas-dotcms** [comment]
This problem happens both on demo (5.2.3) and in the new starter (for the next release).  So we should figure out if it's some kind of corruption that got into the Spanish language on the starter, or if any customers with existing languages might run into the same problem when they upgrade to 5.2.3.

**Comment by john-thomas-dotcms** [comment]
## Console errors

### Error when opening the portlet

Here are the console messages I get when I use Spanish in the Content Search portlet (or any custom content portlet) on demo/starter:

```
--before show--yahoo-- {}
VM15140 VirtualSessionHelper.js:80 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
window.XMLHttpRequest.open @ VM15140 VirtualSessionHelper.js:80
req.getText @ VM15144 dojo.js:2
_32 @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_37 @ VM15144 dojo.js:2
_76 @ VM15144 dojo.js:2
_16 @ VM15144 dojo.js:2
_86 @ VM15144 dojo.js:2
load @ VM15144 dojo.js:2
_e6 @ VM15144 dojo.js:2
_32 @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_37 @ VM15144 dojo.js:2
_76 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
_7a @ VM15144 dojo.js:2
_ef @ VM15144 dojo.js:2
_32 @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_37 @ VM15144 dojo.js:2
_76 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
_7a @ VM15144 dojo.js:2
_ef @ VM15144 dojo.js:2
_32 @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_37 @ VM15144 dojo.js:2
_76 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
_7a @ VM15144 dojo.js:2
_ef @ VM15144 dojo.js:2
_32 @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_37 @ VM15144 dojo.js:2
_76 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
_7a @ VM15144 dojo.js:2
_ef @ VM15144 dojo.js:2
_32 @ VM15144 dojo.js:2
_9 @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
_37 @ VM15144 dojo.js:2
_76 @ VM15144 dojo.js:2
_16 @ VM15144 dojo.js:2
req @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
(anonymous) @ VM15144 dojo.js:2
Show 26 more frames
yahoo.js:8 --before show--yahoo-- {}
layout?p_l_id=71b8a1ca-37b6-4b6e-a43b-c7482f28db6c&p_p_id=content&p_p_action=0&&dm_rlout=1&r=1578504667505&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=content:4427 Uncaught SyntaxError: Unexpected identifier
layout?p_l_id=71b8a1ca-37b6-4b6e-a43b-c7482f28db6c&p_p_id=content&p_p_action=0&&dm_rlout=1&r=1578504667505&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=content:5485 Uncaught ReferenceError: doSearch is not defined
    at Object.onChange (layout?p_l_id=71b8a1ca-37b6-4b6e-a43b-c7482f28db6c&p_p_id=content&p_p_action=0&&dm_rlout=1&r=1578504667505&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=content:5485)
    at Object.<anonymous> (VM15149 build.js:2)
    at VM15144 dojo.js:2
    at Object.<anonymous> (VM15149 build.js:2)
    at VM15144 dojo.js:2
```

### Error when clicking "Advanced" button

When I click on the "Advanced" button on the filter sidebar, I get this in the console:

```
VM15592:1 Uncaught ReferenceError: toggleAdvancedSearchDiv is not defined
    at <anonymous>:1:1
```

### Error when clicking "Clear Search" button

And when I click on the "Clear Search" button ("BUSQUEDA" in Spanish) I get this in the console:

```
Uncaught ReferenceError: clearSearch is not defined
    at Object.eval [as onClick] (eval at construct (VM15149 build.js:2), <anonymous>:3:1)
    at Object._onClick (VM15149 build.js:2)
    at Object._19a (VM15144 dojo.js:2)
    at Object._onClick (VM15149 build.js:2)
    at HTMLSpanElement.<anonymous> (VM15144 dojo.js:2)
    at HTMLSpanElement.<anonymous> (VM15149 build.js:2)
eval @ VM15551:3
_onClick @ build.js?b=5.2.3:2
_19a @ dojo.js?b=5.2.3:2
_onClick @ build.js?b=5.2.3:2
(anonymous) @ dojo.js?b=5.2.3:2
(anonymous) @ build.js?b=5.2.3:2
```

**Comment by john-thomas-dotcms** [comment]
Note: This might be due to some problem with one of the string translations - possibly a string which has a single quote or some other Unicode character in it, which is breaking the JS.

I don't have a way to debug this, but if someone can check, and if this is the problem please let me know which string is causing the problem so I can update the Translations to fix it.

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/17869
PR to fix some translations: https://github.com/dotCMS/core/pull/17900

**Comment by erickgonzalez** [comment]
Some of the Translations that were wrong: 

![Screen Shot 2020-01-23 at 11 40 39 AM](https://user-images.githubusercontent.com/5278970/73026289-c1268e00-3df6-11ea-90e8-4d57ab0988e0.png)

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.5 // Postgres // FF and now we are able to use that portlet without problems in Spanish and other languages
